prismic-model/js/exhibition-resources.js

```diff
@@ -0,0 +1,13 @@
+import label from './types/label';
+import select from './parts/select';
+
+const typeLabel = 'Exhibition resource';
+const labelObject = label(typeLabel);
+const labelObjectWithIcon = {
```

Comment:
The pattern we've used elswhere is to call it what it is i.e. `ExhibitionHighlights`